optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/comparable/TestdataSolutionWithDifficultyComparatorEntity.java

```diff
@@ -0,0 +1,65 @@
+package org.optaplanner.core.impl.testdata.domain.comparable;
+
+import java.util.List;
+
+import org.optaplanner.core.api.domain.solution.PlanningEntityCollectionProperty;
+import org.optaplanner.core.api.domain.solution.PlanningScore;
+import org.optaplanner.core.api.domain.solution.PlanningSolution;
+import org.optaplanner.core.api.domain.solution.drools.ProblemFactCollectionProperty;
+import org.optaplanner.core.api.domain.valuerange.ValueRangeProvider;
+import org.optaplanner.core.api.score.buildin.simple.SimpleScore;
+import org.optaplanner.core.impl.domain.solution.descriptor.SolutionDescriptor;
+import org.optaplanner.core.impl.testdata.domain.TestdataObject;
+import org.optaplanner.core.impl.testdata.domain.TestdataValue;
+
+@PlanningSolution
+public class TestdataSolutionWithDifficultyComparatorEntity extends TestdataObject {
```

Comment:
This naming is very confusing: it ends with "Entity", but it's a planning solution.
This naming should be consistent with it's entity.

Proposal A) for names:
- the solution: TestdataDifficultyComparingSolution
- the entity: TestdataDifficultyComparingEntity